src/codewhisperer/service/inlineCompletionService.ts

```diff
@@ -18,10 +18,11 @@ import { getLogger } from '../../shared/logger/logger'
 import { TelemetryHelper } from '../util/telemetryHelper'
 import { runtimeLanguageContext } from '../util/runtimeLanguageContext'
 import { Commands } from '../../shared/vscode/commands2'
-import { getPrefixSuffixOverlap } from '../util/commonUtil'
+import { getPrefixSuffixOverlap, isInlineCompletionEnabled } from '../util/commonUtil'
 import globals from '../../shared/extensionGlobals'
 import { AuthUtil } from '../util/authUtil'
 import { shared } from '../../shared/utilities/functionUtils'
+import { InlineCompletion } from './inlineCompletion'
```

Comment:
'./inlineCompletion' is a fairly old hacky code that we want to remove. 

 inlineCompletionService should not import from it.

Reply:
If >99% of users are using new inline, you can leave the inlineCompletion.ts unchanged.

Reply:
so we don't want to cover this codepath at all? if so I can remove

Reply:
Yes please leave the inlineCompletion.ts unchanged. 

I did a query and found no one using VSC earlier than 1.68.0 last wk.

Reply:
done